lib/models/instance.js

```diff
@@ -139,6 +138,7 @@ Instance.prototype.listenToSocketEvents = function () {
  * @return {[type]}      [description]
  */
 Instance.prototype.handleSocketData = function (data) {
+  /*jshint maxcomplexity:12 */
```

Comment:
No, this actually only sets the complexity for that function. Tested with my VIM setup and `jshing` CLI tool.